TOCropViewController/Views/TOCropView.m

```diff
@@ -819,6 +819,16 @@ - (CGRect)croppedImageFrame
     frame.size.height = ceilf(cropBoxFrame.size.height * (imageSize.height / contentSize.height));
     frame.size.height = MIN(imageSize.height, frame.size.height);
     
+    // if frame goes beyond boundaries of the image, we move it back
+    // so it is within the boundaries.
+    if (frame.origin.x + frame.size.width > imageSize.width) {
+        frame.origin.x -= frame.origin.x + frame.size.width - imageSize.width;
```

Comment:
@sammy-SC actually, upon reading these lines again, you can do:

```
frame.origin.x = imageSize.width - frame.size.width;
```

And it has the same effect